bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/PerChannelBookieClient.java

```diff
@@ -2288,6 +2288,8 @@ private int statusCodeToExceptionCode(StatusCode status) {
                 return BKException.Code.WriteOnReadOnlyBookieException;
             case ETOOMANYREQUESTS:
                 return BKException.Code.TooManyRequestsException;
+            case EUNKNOWN:
```

Comment:
what about naming this EUNKNOWNLEDGERSTATE ? otherwise people can think that is is kind of "unknown/generic error"

Reply:
Yes I like that, done.